docs/doxygen/logging/logging_architecture.xml

```diff
@@ -0,0 +1 @@
+<mxfile host="drawio.corp.amazon.com" modified="2020-09-11T23:31:12.960Z" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36" etag="9leVIf8lbzMVKf351f_K" version="12.4.8"><diagram id="RtVtmuMCn8x6dfmUY6Zj" name="Page-1">xVfLUtswFP0aL9PxO86SJFCGwkAbKKU7RZZtgWx5ZDmx+/WVLfkVJyltgLJgfI9kPc655/pGsxZx8ZmBNLqhPiKaqfuFZi0105yZhvhfAaUEbE+XQMiwL6EesMK/kASNBs2xjzKFSYhTSjhOhyCkSYIgH2CAMbodTgso8QdACkI0AlYQkDH6iH0eSdRz9A6/RDiMmp0NXY2sAXwJGc0TtZ9mWkH9J4dj0Kyl5mcR8Om2B1nnmrVglHL5FBcLRCpqh7RdHBhtz81Qwl/zQvZwUXhXl872GT7cz4vbiZ79nFhylQ0gOWquUR+Wlw1B2whztEoBrOKtyAHNmkc8JiIyxCPIUilLgAsktpr7IIuqh3o0wIQsKKGsXsuCMAggFLjaFjGOioP3MVqWRPIhGiPOSjFFvWB66qwq8RxbhttORdNV3Ec9BVsQqMwJ26U79sSDInA/mV83XvnoullcPE23369u8tvVt4npjchDvkg2FVLGIxrSBJDzDp3DnG1aurop15SmCnxGnJfKOCDndMh/xgHjZ5URBJDQBDXYBa5Ov9RFjBK/mbEmFL7cRziRsJpUS0UTrnYxjFaikR7iejRnEB3hwXCVjwELET82cSYnViwdlZwhAjjeDC375vo15+654RqvGWDlpEpyHGBYFaI84zSWFSnA4VhyQkTtqmT4g3N2Cd/xylpHFnJrNRl9Qb0RHXm65x2TaGSsgy6ydGPgIsOYjmxk6NbYRo3dTlFhb0n6HyYS/LHyR+sWETxVwSenCZdFf3BZqmhv+v6jaaavNY17okfqV0U9AGVvQkpxwrPeyncV0OWJMfOGeWLbO0rLFTvd26OdYMjpyJCLyWr5RUDKmO/mPvEdN+sv1ch9vrt2Hfdt3Gc4zoBVUzc/zn1HS3KP8juRVwFlcb8IXtMwxIkofvqKiz5Ia5ucd1AC+DPf2acEQLZhmW+jhG26O0roe5SwP7AO2uPcr788iAn0LE0JhsLvNBkRL27Md3uEPnOqSejTrCBAcCgagyUUPIptrHnFn9iGnKmBGPs+OSRp3QvXVbgrjSd1eFbDrdLEdceSmPs6vL9v8ETYdeKyfHW/dqzz3w==</diagram></mxfile>
```

Comment:
This doesn't need to be compressed to base64. You can just check in the original xml.

Reply:
Fixed